packages/components/src/components/tabs/component.tsx

```diff
@@ -49,25 +48,22 @@ export class KolTabs implements API {
 	};
 
 	private onKeyDown = (event: KeyboardEvent) => {
-		const timeout = setTimeout(() => {
```

Comment:
Das Entfernen der Timeouts behebt hier das Problem, daher würde ich es als naheliegende Lösung anbieten.

Ich gehe davon aus, dass die Timeouts einen Sinn hatten, aber ich konnte ihn nicht nachvollziehen. Zum Nutzen habe ich bei meiner kurzen Recherche nichts finden können. Die in unserer Readme verlinkten Beispiele kommen allesamt ohne aus.
Mein persönliches Empfinden ist, dass die Komponente jetzt nicht mehr so träge wirkt und sich besser bedienen lässt.

Reply:
Okay